pages/locations/index.js

```diff
@@ -100,6 +115,7 @@ export default class LocationsPage extends Component {
               queryString={queryString}
             />
             <LocationList locations={filteredLocations} />
+            <button onClick={this.loadMore}>Load More</button>
```

Comment:
Is there a reason we do this instead of all of them? Maybe we should just lazily load them after render.